web/src/components/njwds-extended/Button.tsx

```diff
@@ -33,19 +35,19 @@ export const Button = (props: Props): ReactElement => {
   switch (props.style) {
     case "primary":
       style = "usa-button padding-y-1";
-      disabledClass = "usa-button--disabled";
+      disabledClass = " usa-button--disabled";
       break;
     case "primary-input-field-height":
       style = "usa-button padding-y-2";
-      disabledClass = "usa-button--disabled padding-y-2";
+      disabledClass = " usa-button--disabled padding-y-2";
       break;
     case "secondary":
       style = "usa-button usa-button--outline padding-y-1";
-      disabledClass = "usa-button--outline-disabled";
+      disabledClass = " usa-button--outline-disabled";
       break;
     case "secondary-input-field-height":
       style = "usa-button usa-button--outline padding-y-2";
-      disabledClass = "usa-button--outline-disabled padding-y-2";
+      disabledClass = " usa-button--outline-disabled padding-y-2";
```

Comment:
why are we adding back the prefix space? we should be doing this in the template string, not in these variables